packages/replay/src/coreHandlers/util/addBreadcrumbEvent.ts

```diff
@@ -27,7 +27,7 @@ export function addBreadcrumbEvent(replay: ReplayContainer, breadcrumb: Breadcru
       timestamp: (breadcrumb.timestamp || 0) * 1000,
       data: {
         tag: 'breadcrumb',
-        payload: normalize(breadcrumb),
```

Comment:
We could use the `normalizeDepth` client option and set the max breadth to smth like 1_000 - which I find a sane number that will prevent an insane amount of data from being generated.

Reply:
Hmm, I think we may want to have a dedicated setting for the depth here eventually, as we may not need as much depth here as in other places, for example. But a good point, will think about it. Will also add 1_000 as max properties, sounds like a good additional check!